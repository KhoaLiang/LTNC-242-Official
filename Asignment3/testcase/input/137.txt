INSERT x number
INSERT y string
BEGIN
INSERT z number
BEGIN
INSERT w string
RPRINT
END
END