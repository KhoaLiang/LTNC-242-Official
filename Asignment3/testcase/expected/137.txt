success
success
success
success
w//2 z//1 y//0 x//0